docs/dev/building.md

```diff
@@ -138,41 +166,41 @@ This section lists all the artifacts generated by the build system.
 
 **Executables**:
 
-* `build/<mode>/scylla`
+* `build/<config>/scylla`
 
 **Tests**:
 
-* `build/<mode>/test/**`
+* `build/<config>/test/**`
```

Comment:
This doesn't seem correct - I see for example build/test/boost/Dev/sstable_datafile_test - the `<config>` is **not** a top-level subdirectory of build. Which I am sad about, because I was used to doing "rm -r build/debug" to delete an unwanted build mode, and I can no longer do that.

Reply:
indeed. should be fixed in the latest revision.